src/main/java/seedu/address/logic/commands/DeleteModuleCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.Module;
+
+/**
+ * Deletes a module identified using it's module code.
+ */
+public class DeleteModuleCommand extends Command {
+
+    public static final String COMMAND_WORD = "module";
+    public static final String DELETE_MODEL = "delete";
```

Comment:
should import these 2 from CommandWords rather than making a new string

Also, should create a single command_word e.g `public static final String COMMAND_WORD = CommandWords.MODULE+ " " + CommandWords.DELETE_MODEL;
`